### PR TITLE
[IGNORE] Update UI dependencies

### PR DESCRIPTION
**Pull request by Gladorme**
Updating UI dependencies => it will remove warnings when building UI (turbopack + 1 high severity vulnerability) 🧹
I think I will do a PR for each package next, I tried to do all of them together, but it's a bad idea... 😅 

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

### CODE REVIEW

File: ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ function createMarkdownPanelStyles(theme: Theme, chartsTheme: PersesChartsTheme)
     height: '100%',
     overflowY: 'auto',
     // Ignore top margin on the first element.
-    '& :first-child': {
+    '& :first-of-type': {
```
Comment (Gladorme) [inline ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx:31]: I got a console error when opening a dashboard:
```
The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".
```
We don't use SSSR, so it's a little bit strange... I need to check if it's the right solution or if we can't delete this property.

Comment (sjcobb) [reply ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx:31]: We've seen a similar error with InfoTooltip before too, I'll try to find that fix

Comment (sjcobb) [reply ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx:31]: Ah nvm, this is [the PR](https://github.com/perses/perses/pull/1068/files#r1156577532) I was thinking of

Hopefully what you did fixes it 🤞, always nice to reduce some console noise

**Comment by sjcobb** [comment]
Got react-router updated finally in #1209!

**Comment by sjcobb** [comment]
> Got react-router updated finally in #1209!

Oh spoke too soon, seeing some warnings actually after testing more

image

**Comment by Gladorme** [comment]
> Got react-router updated finally in #1209!

Cool! 🚀 

> Oh spoke too soon, seeing some warnings actually after testing more

I will check 🥲

File: ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx (hunk 1 above)
Comment (Gladorme) [reply ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx:31]: I did some check, the fix is working fine 😄

**Comment by Gladorme** [comment]
PR ready, however `ci / build-storybook` seems to not start

**Comment by sjcobb** [comment]
Hey @Gladorme do you remember if running ‘npm run storybook’ still worked locally and you could see the stories? Trying to figure out if this was a weird CI issue or an actual issue with Storybook and the updated versions

**Comment by Gladorme** [comment]
> Hey @Gladorme do you remember if running ‘npm run storybook’ still worked locally and you could see the stories? Trying to figure out if this was a weird CI issue or an actual issue with Storybook and the updated versions

Hi @sjcobb, it didn't work locally neither. I fear the issue is related to the updated versions :/

**Comment by sjcobb** [comment]
Looked into rebasing this branch in #1333 
The Storybook tests hanging in CI is a known node 18 issue that has a workaround we can try out 🤞 see comment here: https://github.com/perses/perses/pull/1333#issuecomment-1652837977

**Comment by Gladorme** [comment]
Changes merged on PR: https://github.com/perses/perses/pull/1333

**Pull request closed without merging**